src/react-components/profile-entry-panel.js

```diff
@@ -9,10 +9,9 @@ import { SCHEMA } from "../storage/store";
 import styles from "../assets/stylesheets/profile.scss";
 import hubLogo from "../assets/images/hub-preview-white.png";
 import { WithHoverSound } from "./wrap-with-audio";
-import { AVATAR_TYPES, getAvatarGltfUrl, getAvatarType } from "../utils/avatar-utils";
+import { fetchAvatar } from "../utils/avatar-utils";
```

Comment:
Nice. I see what you mean now.
Looks like we can get rid of `getAvatarGltfUrl` now. Nothing else uses it.